models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R

```diff
@@ -58,8 +58,8 @@ AddSetting(name="sigma", 		   comment='surface tension')
 # 	Inputs: Fluid Properties
 AddSetting(name="omega_l", comment='one over relaxation time (low density fluid)')
 AddSetting(name="omega_h", comment='one over relaxation time (high density fluid)')
-AddSetting(name="nu_l", omega_l='1.0/(3*nu_l)', default=0.16666666, comment='kinematic viscosity')
-AddSetting(name="nu_h", omega_h='1.0/(3*nu_h)', default=0.16666666, comment='kinematic viscosity')
+AddSetting(name="Viscosity_l", omega_l='1.0/(3*Viscosity_l)', default=0.33333333, comment='kinematic viscosity')
+AddSetting(name="Viscosity_h", omega_h='1.0/(3*Viscosity_h)', default=0.33333333, comment='kinematic viscosity')
 AddSetting(name="S0", default=1.0, comment='Relaxation Param')
 AddSetting(name="S1", default=1.0, comment='Relaxation Param')
 AddSetting(name="S2", default=1.0, comment='Relaxation Param')
```

Comment:
Just confirming this is corresponding to the relaxation parameters? I have not played with adjusting any relaxation parameters - had just been my initial set up for the implementation. I will comment them out in the update commit.